src/postprocessor.rs

```diff
@@ -59,6 +60,7 @@ pub trait Postprocessor: Send + Sync {
 
 pub fn lookup_with_config(config: &PostprocessorConfig) -> Result<Box<dyn Postprocessor>> {
     match config.name.as_str() {
+        "chunk" | "chonk" => Ok(Box::new(chunk::Chunk::from_config(config.config.as_ref())?)),
```

Comment:
As much as I love the `chonk` name, I think having two might lead to unnecessary confusion and less grepability, so I'd stick to just `chunk`